docs/testing/unit-testing.md

```diff
@@ -54,21 +38,21 @@ export function add(x, y) {
 }
 ```
 
-> Note: The `export` here is ES6 syntax, and you will need an ES6 transpiler
+> Note: The `export` here is ES6 syntax, and you would need an ES6 transpiler
   (e.g. babel.js) to run this JavaScript.
 
-> The `export` exports our function as a module, which we can `import` and use
+> The `export` makes our function available as a module, which we can `import` and use
   in other files. Continue below to see what that looks like.
 
-### Mocha
+### Jest
 
-Mocha is our unit testing framework. Its API, which we write tests with, is
+Jest is our unit testing framework. Its API, which we write tests with, is
 speech like and easy to use.
 
-> Note: This is the [official documentation](http://mochajs.org) of Mocha.
+> Note: The official documentation for Jest can be found [here](https://facebook.github.io/jest/).
 
 We're going to add a second file called `add.test.js` with our unit tests
-inside. Running said unit tests requires us to enter `mocha add.test.js` into
+inside. Running said unit tests requires us to enter `jest add.test.js` on
 the command line.
```

Comment:
I have an issue with 

> Running said unit tests requires us to enter `jest add.test.js` on
  the command line.

This command won't actually run. The correct command would be something ugly like

`node ./node_modules/jest-cli/bin/jest app/components/Button/tests/index.test.js`

I'm doubtful that it would be a good idea to add that full command. One alternative is to follow the Jest install instruction in Jest's [Getting Started Guide](https://facebook.github.io/jest/#getting-started)

Given that Jest is already installed in RBP and that no mention is made of actually running Jest in `docs/testing/component-testing.md`, I'm inclined t just remove this line.

What do you think?